### ISSUE TITLE
Links

### ISSUE THREAD

**Issue by caltrop-3** [tomboy-notes/tomboy#62]
ANNOYING having links create themselves (to other notes) while typing.

Need to be a way to turn that OFF!

**Comment by ajhepple** [comment]
Hi @caltrop-3, thanks for your feedback.

Automatic wiki-style linking between notes is Tomboy's primary and probably most loved feature. Together with the fact that Tomboy aims to keep things simple and minimise configuration options, I'm not sure that there is support for an 'OFF' switch.

However, there have been requests to control auto-linking in the past. As far as I recall, discussions usually conclude that some sort of custom title strategy is the best way forward for users who dislike the default behaviour. For example, including a time or date in note titles greatly reduces the likelihood of them being linked to automatically. I believe that using camel case for titles is also effective.  Perhaps you can use one of these strategies?

**Comment by caltrop-3** [comment]
Yeah, ,made my titles so bizarre that there's no way a link will POP-UP while typing.
Have the same issue in Office Packages, M$ Office & Libre Office.
I just hate a link making itself unless I specifically want it.
Thanks anyway.

**Comment by alex-ter** [comment]
As @ajhepple said, this was built-in from the very beginning and as such is quite entrenched, both in the code and user's perception (the latter is my guess). However I don't why why it can't be made an option (enabled by default). So let me classify this as a feature request and if someone is willing to provide a patch - go ahead and open a pull request.
